tests/test_ds.py

```diff
@@ -13,29 +14,42 @@ def test_load_file():
 
     single_cube = ds.DagmcStats(test_env[1]['input_file'])
     assert(single_cube.root_set == 0)
-    
+
     # tests for pandas data frames
     exp_vert_data = pd.DataFrame(
-            columns=['vert_eh', 'roughness', 'tri_per_vert'])
+        columns=['vert_eh', 'roughness', 'tri_per_vert'])
     exp_vert_data = exp_vert_data.set_index('vert_eh')
     assert(single_cube._vert_data.equals(exp_vert_data))
-    
+
     exp_tri_data = pd.DataFrame(
-            columns=['tri_eh', 'aspect_ratio', 'area'])
+        columns=['tri_eh', 'aspect_ratio', 'area'])
     exp_tri_data = exp_tri_data.set_index('tri_eh')
     assert(single_cube._tri_data.equals(exp_tri_data))
-    
+
     exp_surf_data = pd.DataFrame(
-            columns=['surf_eh', 'tri_per_surf', 'coarseness'])
+        columns=['surf_eh', 'tri_per_surf', 'coarseness'])
     exp_surf_data = exp_surf_data.set_index('surf_eh')
     assert(single_cube._surf_data.equals(exp_surf_data))
-    
+
     exp_vol_data = pd.DataFrame(
-            columns=['vol_eh', 'surf_per_vol', 'coarseness'])
+        columns=['vol_eh', 'surf_per_vol', 'coarseness'])
     exp_vol_data = exp_vol_data.set_index('vol_eh')
     assert(single_cube._vol_data.equals(exp_vol_data))
 
 
 def test_load_file_populate():
     # another test to test load_file() with populate=True
     pass
+
+
+def test_native_ranges():
+    """Tests get_native_ranges
+    """
+    single_cube = ds.DagmcStats(test_env[1]['input_file'])
+    meshset = single_cube.root_set
+    native_ranges = single_cube._DagmcStats__set_native_ranges(entity_types)
```

Comment:
```suggestion
    native_ranges = single_cube._DagmcStats__set_native_ranges()
```
for consistency with changing to a class variable

Reply:
actually, since this is now called in the constructor we don't need to explicitly call it here and can just test that it is set properly in the constructor.